api/src/main/java/org/apache/iceberg/actions/RewriteDataFiles.java

```diff
@@ -199,6 +203,10 @@ interface FileGroupRewriteResult {
     int addedDataFilesCount();
 
     int rewrittenDataFilesCount();
+
+    default long rewrittenBytesCount() {
```

Comment:
Since we are modifying the API here we should probably also add some tests (maybe just 1) to TestRewriteDataFilesAction test classes (not just the procedure)

Reply:
done